contracts/staking/contracts/src/staking_pools/MixinStakingPool.sol

```diff
@@ -276,7 +141,7 @@ contract MixinStakingPool is
         pure
         returns (bytes32)
     {
-        return bytes32(uint256(poolId).safeAdd(POOL_ID_INCREMENT_AMOUNT));
+        return bytes32(uint256(poolId).safeAdd(1));
```

Comment:
This can probably be inlined in `createStakingPool` (also see my comment on `nextPoolId`)